kategory-docs/docs/docs/datatypes/ior/README.md

```diff
@@ -6,4 +6,125 @@ permalink: /docs/datatypes/ior/
 
 ## Ior 
 
-TODO
\ No newline at end of file
+`Ior` represents an inclusive-or relationship between two data types.
+This makes it very similar to the [`Either`](/docs/datatypes/either) data type, which represents an "exclusive-or" relationship.
+What this means, is that an `Ior<A, B>` (also written as `A Ior B`) can contain either an `A`, a `B`, or both an `A` and `B`.
+Another similarity to `Either` is that `Ior` is right-biased,
+which means that the `map` and `flatMap` functions will work on the right side of the `Ior`, in our case the `B` value.
+You can see this in the function signature of `map`:
+
+```kotlin
+fun <D> map(f: (B) -> D): Ior<A, D>
+```
+
+We can create `Ior` values using `Ior.left`, `Ior.right` and `Ior.both`:
+
+```kotlin:ank
+import kategory.*
+
+val right = Ior.Right(3)
```

Comment:
for ank to print the result of this expressions they should be added into separate snippets. Ank can only evaluate a snippet as a whole.